src/components/Form/Form.js

```diff
@@ -0,0 +1,34 @@
+import useStore from '../../hooks/useStore';
+
+import StyledForm from './StyledForm';
+
+export default function Form() {
+	const addTask = useStore(state => state.addTask);
+
+	function handleSubmit(event) {
+		event.preventDefault();
+		const form = event.target;
+		const name = form.elements.task.value.trim();
+		const duration = Number.parseInt(form.elements.duration.value, 10);
+		addTask(name, duration);
+		form.reset();
+	}
+
+	return (
+		<StyledForm onSubmit={handleSubmit} name="task">
```

Comment:
Do you need the `name` attribute on the form?

Reply:
Yes, the form geht her role only when it have a name. I use getByRole in the first test.